templates/CRM/Financial/Form/Search.tpl

```diff
@@ -9,6 +9,9 @@
 *}
 
 {* Financial search component. *}
+{if !isset($batchStatus)}
+  {assign var="batchStatus" value="open"}
+{/if}
```

Comment:
I'm just going to double-check this one to see if it prevents "any" as the default.

Otherwise I think everything else is ok.

Reply:
Seems ok. This is for the access key, although I'm not sure what the `context` would mean there. Anyway...